server/handler.go

```diff
@@ -128,6 +128,11 @@ func (s *Server) LoginHandlerFunc(w http.ResponseWriter, r *http.Request) {
 	user := r.FormValue("user")
 	password := r.FormValue("password")
 
+	err = validateRequest(r)
+	if err != nil {
+		return
```

Comment:
It's better to return error in case when exec result can be different for `LoginHandlerFunc`, or at least log that error

Reply:
ok